core/src/Kokkos_HIP_Space.hpp

```diff
@@ -277,6 +364,42 @@ struct MemorySpaceAccess<Kokkos::Experimental::HIPHostPinnedSpace,
   enum : bool { deepcopy = true };
 };
 
+template <>
+struct MemorySpaceAccess<Kokkos::Experimental::HIPHostPinnedSpace,
+                         Kokkos::Experimental::HIPManagedSpace> {
+  enum : bool { assignable = false };  // different exec_space
+  enum : bool { accessible = true };
+  enum : bool { deepcopy = true };
+};
+
+//----------------------------------------
+// HIPManagedSpace::execution_space != HostSpace::execution_space
+// HIPManagedSpace accessible to both HIP and Host
+
+template <>
+struct MemorySpaceAccess<Kokkos::Experimental::HIPManagedSpace,
+                         Kokkos::HostSpace> {
+  enum : bool { assignable = false };  // Cannot access from HIP
+  enum : bool { accessible = false };  // HIPHostPinnedSpace::execution_space
```

Comment:
The comment is wrong

Reply:
deleted in [145ccf](https://github.com/kokkos/kokkos/pull/5112/commits/145ccf2bb360efd1d1272410afc9386c8cbb3878)